src/main/java/com/opentable/service/ServiceInfo.java

```diff
@@ -14,11 +14,151 @@
 
 package com.opentable.service;
 
+import java.net.URI;
+import java.util.Objects;
+
 /**
  * The {@link ServiceInfo} interface is intended for you to declare as a Spring Bean in your application.
  * It can be consumed by our components in order to create things like metric namespaces given information about your
  * service.  You are expected to implement your own implementations of this interface when defining your Bean.
+ * The easiest way to do so is use the Builder.
+ *
+ * For more details on these fields see https://wiki.otcorp.opentable.com/display/CP/The+service-details+Endpoint
  */
 public interface ServiceInfo {
+    /**
+     * Service Name
+     */
     String getName();
+
+    /**
+     * A description of the service
+     */
+    String description();
+
+    /**
+     * The name of the team responsible for the operation of the service.
+     */
+    String teamName();
+
+    /**
+     * The email of the team
+     */
+    String teamEmail();
+
+    /**
+     * PagerDuty contact info
+     */
+    String pagerDutyContact();
+
+    /**
+     * The github repo wherein the service code resides
+     */
+    URI repository();
+
+    /**
+     * Link to documentation, typically a wiki or README.MD
+     */
+    URI documentation();
+
+    /**
+     * Link to the runbook, typically in the wiki
+     */
+    URI runBook();
+
+    /**
+     * Link to the metrics page in Graphite
+     */
+    URI metrics();
+
+    /**
+     * Link to the kibana dashboard
+     */
+    URI logs();
```

Comment:
We could provide a default here based on the service name.  Some services have not built out extensive saved Kibana dashboards due to the difficulty of managing them there.

Further, here too, like for Grafana dashboards, service authors could have _multiple_ Kibana dashboards.